packages/stores/types.ts

```diff
@@ -14,6 +14,7 @@ import type {
     SetFinancialAssessmentRequest,
     SetFinancialAssessmentResponse,
     StatesList,
+    WebsiteStatus,
 } from '@deriv/api-types';
 import type { Moment } from 'moment';
```

Comment:
need to change the order of imports here

Reply:
Not part of account package